libs/design-system-components/src/components/ExtensionEditStep3Form/index.tsx

```diff
@@ -149,14 +150,16 @@ const ExtensionEditStep3Form: React.FC<ExtensionEditStep3FormProps> = props => {
     event.preventDefault();
     const formValues = getValues();
 
-    formValues.contributors = addedContributors?.map(profile => profile?.did?.id);
-    formValues.keywords = [...keywords]?.filter(keyword => keyword);
-
     if (formValues.license === Licenses.OTHER) {
       formValues.license = formValues.licenseOther;
     }
+
     if (isValid) {
-      nextButton.handleClick(formValues);
+      nextButton.handleClick({
+        ...formValues,
+        contributors: addedContributors?.map(profile => profile?.did?.id),
+        keywords: [...keywords]?.filter(keyword => keyword),
```

Comment:
is this filter to remove possible `null`, `undefined` or `false` values?

Reply:
Yes, thats my understanding as well.  Actually this filter was there before this PR. There is an array of available keywords the component will receive and add it to the current keywords set as the user selects one.